frameworks/kafka/tests/auth.py

```diff
@@ -42,22 +42,6 @@ def get_service_principals(service_name: str, realm: str, custom_domain: str = N
     return principals
 
 
-def wait_for_brokers(client: str, brokers: list):
-    """
-    Run bootstrap on the specified client to resolve the list of brokers
-    """
-    LOG.info("Running bootstrap to wait for DNS resolution")
-    bootstrap_cmd = ['/opt/bootstrap',
```

Comment:
There is a subtle difference between `wait_for_brokers` and `resolve_hosts` the Kafka client is a docker image that has `bootstrap` copied to `/opt/bootstrap`. Since the PR passes, maybe that's ok?

Reply:
Looking at the build output, this doesn't cause the tests to fail as expected:
```
[2018-03-27 03:45:59,252|sdk_cmd|INFO]: STDERR:
Failed to execute './bootstrap': No such file or directory
[2018-03-27 03:45:59,253|sdk_cmd|WARNING]: Could not resolve: kafka-0-broker.kafka.autoip.dcos.thisdcos.directory
[2018-03-27 03:45:59,253|sdk_cmd|WARNING]: Could not resolve: kafka-1-broker.kafka.autoip.dcos.thisdcos.directory
[2018-03-27 03:45:59,253|sdk_cmd|WARNING]: Could not resolve: kafka-2-broker.kafka.autoip.dcos.thisdcos.directory
[2018-03-27 03:45:59,253|sdk_cmd|INFO]: (CLI) dcos kafka --name=kafka topic create authn.test
```
I will address this when resolving conflicts for #2419